atuin-client/Cargo.toml

```diff
@@ -52,6 +52,7 @@ thiserror = { workspace = true }
 futures = "0.3"
 crypto_secretbox = "0.1.1"
 generic-array = { version = "0.14", features = ["serde"] }
+crossterm = { workspace = true }
```

Comment:
I'd really like to avoid adding the crossterm dependency here. Where possible, keeping UI/interaction style dependencies to the `atuin` crate is desireable. Given that it's only needed for the `SetCursorStyle` enum, would it be possible to remove this?

Reply:
OK. It's possible by giving up the conversion through the `From<>` trait. I'll update it.

Reply:
In commit 15cbded2e0831c6aaa3d479d8ba0a91ce7501e50, I moved the conversion function into `atuin` and removed the dependency from `atuin-client`. Could you check it?

Reply:
looks good, thanks!